### PR TITLE
fix: update guide article for javascript conditional ternary operator

### LINKED ISSUE #35386: Error in solution for "Basic JavaScript: Use the Conditional (Ternary) Operator"
The solution for "Basic Javascript: Use the Conditional (Ternary) Operator", linked below, contains a an error:

Challenge: https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/basic-javascript/use-the-conditional-ternary-operator

Solution: https://guide.freecodecamp.org/certifications/javascript-algorithms-and-data-structures/basic-javascript/use-the-conditional-ternary-operator/

Indeed, the working solution is not: return (a = b ? true : false ).
It's return (a == b ? true : false ).

### PR DESCRIPTION
**Pull request by rupali317**
For the challenge "Basic JavaScript: Use the Conditional (Ternary) Operator", if the expression evaluates to true then we should return "Equal", otherwise we return "Not Equal"

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #35386

### CODE REVIEW

File: guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/use-the-conditional-ternary-operator/index.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Remember that the "traditional" `if...else` syntax can be re-written using the c
 
 ```javascript
 function checkEqual(a, b) {
-  return a === b ? true : false;
+  return a === b ? "Equal" : "Not Equal";
 }
 ```
 ·  Run code at [repl.it](https://repl.it/@AdrianSkar/Basic-JS-Ternary-operator).
```
Comment (thecodingaviator) [inline guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/use-the-conditional-ternary-operator/index.md]: You'd also need to do something about this

Comment (rupali317) [reply guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/use-the-conditional-ternary-operator/index.md]: @thecodingaviator, want to confirm that changing the content in that repl.it link and updating the link with https://repl.it/@rupali317/Basic-JS-Ternary-operator is fine right?

Comment (rupali317) [reply guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/use-the-conditional-ternary-operator/index.md]: I made the change

**Comment by RandellDawson** [comment]
Hold off on merging this one until we get some direction in general about adding repl.it links.  We are currently discussing if Guide articles for the curriculum challenges should even have these types of external links (which we do not control).  If someone deletes their repl account or accidentally deletes the session, then we have a 404 on our hands.

**Comment by Nitin96Bisht** [comment]
@RandellDawson Instead we can use shared link which does not require any repl account like this  https://repl.it/repls/SubduedSteelblueAttribute

**Comment by Bouncey** [comment]
We do not need repl links in challenge guide articles. The user presumably is already on the challenge concerned, and any code related to the solution should be runnable in the challenge editor directly.

**Comment by rupali317** [comment]
I removed the repl.it link from the solution @Bouncey

**Review by RandellDawson — APPROVED** [review]
@thecodingaviator I had to resolve some merge conflicts due to me merging PR #34310 before this one. 
 
I made one small update that was left over from the previous PR regarding the 3rd bullet in the Code Explanation section.  You can see it in the changes.

I believe this final version looks good.  If you are OK with the changes, then merge it.

**Comment by thecodingaviator** [comment]
Thanks @rupali317 for contributing to this repo! 👍 

We're glad to accept your changes and look forward to more contributions from you!

**Pull request merged**